pom.xml

```diff
@@ -95,12 +96,6 @@
             <version>${thymeleaf.version}</version>
         </dependency>
 
-        <dependency>
```

Comment:
Removal of JUnit makes the project unbuildable (except when using Thorntail profile)

I run project locally on Payara which is no longer possible.

Reply:
Hmm. Lemme check on it.